convertsortedlisttobinarysearchtree/convert-sorted-list-to-binary-search-tree.js

```diff
@@ -0,0 +1,29 @@
+// https://leetcode.com/problems/convert-sorted-list-to-binary-search-tree/ 
+function convertSortedListToBinarySearchTree (head) {
+    const brinaryTree = []
+    const arrayPositiveNumb = []
+    const arrayNegativeNumb = []
+    for (const numb of head){
+            if(numb === 0){
+            brinaryTree.push(numb)
+        } else if (numb > 0){
+            arrayPositiveNumb.push(numb)
+            arrayPositiveNumb.sort().reverse()
```

Comment:
moze by z tego funkcje zrobic bo duplikacja kodu jest, bo to samo masz w else if pod spodem